lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -318,6 +339,11 @@ sub run_java_vm {
 sub create {
     my $class = shift;
     my $self = $class->SUPER::create(@_);
+
+    if (not defined $self->use_version) {
+        die $self->error_message('Cannot pass undef to Picard::use_version');
```

Comment:
This comment is orthogonal to your comment above: Rather than putting this check in create, you can make it non-optional, right?